lib/apartment/adapters/postgresql_adapter.rb

```diff
@@ -15,6 +15,15 @@ module Adapters
     # Default adapter when not using Postgresql Schemas
     class PostgresqlAdapter < AbstractAdapter
 
+    def initialize(config)
+      super
+
+      Apartment.connection_handler.clear_all_connections! # clear out old connection pools
```

Comment:
Without this line, `ActiveRecord::Base.connection` calls outside of apartment (like when checking for pending migrations) would eventually raise `ActiveRecord::ConnectionTimeoutError`s.